gsa/src/web/entity/__tests__/box.js

```diff
@@ -17,22 +17,36 @@
  */
 import React from 'react';
 
-import Date, {setLocale} from 'gmp/models/date';
+import date, {setLocale} from 'gmp/models/date';
 
-import {render} from 'web/utils/testing';
+import {rendererWith} from 'web/utils/testing';
+import {setTimezone, setUsername} from 'web/store/usersettings/actions';
 
 import EntityBox from '../box';
 
 setLocale('en');
 
-const date = Date('2019-01-01T12:00:00Z');
-const date2 = Date('2019-02-02T12:00:00Z');
+const manualUrl = 'test/';
```

Comment:
Do we really need the manualUrl here? I can't figure out where that would be used.

Reply:
It's used in the help icon to direct to the manual. Normally it's set in GOS, but here we probably need it for testing to be test...

Reply:
I'm sorry, I should have phrased my question better. This is the test for the entity box. Where do we use a help icon here? I can't find any manual links here and I don't think we need the manualUrl for anything else.

And as a note: for tests with a help icon (or rather with a manual link) the manualUrl needs to be set but it doesn't have to be `'test/'` it just needs to be defined.